### PR TITLE
Enable Sorting by Metadata Name in MeshSync Resources

### PR DESCRIPTION
**Pull request by singh1203**
**Notes for Reviewers**
Update the order handling logic to include kubernetes_resource_meta.name as a sortable field.

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: server/handlers/meshsync_handler.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func (h *Handler) GetMeshSyncResources(rw http.ResponseWriter, r *http.Request,
 		result = result.Offset(offset)
 	}
 
-	order = models.SanitizeOrderInput(order, []string{"created_at", "updated_at", "name"})
+	order = models.SanitizeOrderInput(order, []string{"created_at", "updated_at", "kubernetes_resource_object_meta.name"})
```
Comment (sudhanshutech) [inline server/handlers/meshsync_handler.go:116]: @singh1203 the request is going for sort from client but its not sorting the data.

Comment (singh1203) [reply server/handlers/meshsync_handler.go:116]: @MUzairS15 can you help me with what I am missing here a request for sort is successful but the sort on the server is not reflecting on the client side.

Comment (MUzairS15) [reply server/handlers/meshsync_handler.go:116]: Try to see what SQL query is getting produced using form debug / ToSQL.
What is the value for `order` after SanitizeOrderInput?

**Comment by Ashparshp** [comment]
@singh1203 Let's discuss this issue during the Meshery dev meeting call. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by sudhanshutech** [comment]
any new updates here @singh1203 ?

**Comment by singh1203** [comment]
> any new updates here @singh1203 ?

@sudhanshutech There hasn't been much progress here, but all I know is that the query responsible for sorting is not formatting well. I am trying my best to find the cause.

**Pull request closed without merging**